src/constants.js

```diff
@@ -0,0 +1,60 @@
+export const TASKS_MANAGEMENT_MAIN_MENU_CONTRIBUTION_KEY = 'tasksManagement.MainMenu';
+
+export const DEFAULT_DEBOUNCE_TIME = 500;
+export const DEFAULT_PAGE_SIZE = 10;
+export const EMPTY_STRING = '';
+export const HYPHEN = '-';
+export const ROWS_PER_PAGE_OPTIONS = [10, 20, 50, 100];
+export const CONTAINS_LOOKUP = 'Icontains';
+
+export const TASK_STATUS = {
+  RECEIVED: 'RECEIVED',
+  ACCEPTED: 'ACCEPTED',
+  COMPLETED: 'COMPLETED',
+  FAILED: 'FAILED',
+};
+
+export const POLICY_STATUS = {
```

Comment:
Please rename it to something like GROUP_RESOLVE_POLICY. POLICY_STATUS could be confused with statuses in policy_js modules

Reply:
Done.